tests/projects/SelfContained_Trimming_Test/Program.fs

```diff
@@ -9123,9 +9159,228 @@ module PercentAInternalTests =
         testPercentA ()
         testPercentPlusA ()
 
+module ClassWithEvents =
+
+    type EventClass () =
+        let event = new Event<string> ()
+
+        [<CLIEvent>]
+        member _.Event = event.Publish
+
+        member _.TriggerEvent (arg) = event.Trigger (arg)
+
+    let testWithEventClass () =
+        test "test9200" (
+            lazy (
+                let mutable eventstring = ""
+                let evt = EventClass()
+                evt.Event.Add(fun arg -> eventstring <- eventstring + $"Event handler: {arg}")
+                evt.TriggerEvent("Hello World!")
+                eventstring
+            ))
+            """Event handler: Hello World!"""
+        test "test9201" (
+            lazy (
+                let mutable eventstring = ""
+                let evt = EventClass();
+                evt.Event.Add(fun arg -> eventstring <- eventstring + $"First handler: {arg}")
+                evt.Event.Add(fun arg -> eventstring <- eventstring + $" Second handler: {arg}")
+                evt.TriggerEvent("Hello World!")
+                eventstring
+            ))
+            """First handler: Hello World! Second handler: Hello World!"""
+    let tests () =
+        testWithEventClass ()
+
+module SameTestsUsingNonStructuralComparison2 =
+    open NonStructuralComparison
+
+    do check "ffcenonoiwe1" (3 > 1) true
+    do check "ffcenonoiwe2" (3y > 1y) true
+    do check "ffcenonoiwe3" (3uy > 1uy) true
+    do check "ffcenonoiwe4" (3s > 1s) true
+    do check "ffcenonoiwe5" (3us > 1us) true
+    do check "ffcenonoiwe6" (3 > 1) true
+    do check "ffcenonoiwe7" (3u > 1u) true
+    do check "ffcenonoiwe8" (3L > 1L) true
+    do check "ffcenonoiwe9" (3UL > 1UL) true
+    do check "ffcenonoiwe9" (3.14 > 3.1) true
```

Comment:
This and elsewhere - I guess the purpose of the string is to be **unique** so that a test failure leads to a single line of test code?

Reply:
It doesn't matter, it's not the real test, we are not running them really, we only compile and trim those.

Reply:
@vzarytovskii , I don't really mind fixing this vlad, it's probably better not to copy and paste test debt :-)

Reply:
renamed them all.